portal-web/test/functional/com/liferay/portalweb/macros/CookieConsentAdmin.macro

```diff
@@ -0,0 +1,139 @@
+definition {
+
+	macro gotoInstanceSettings {
```

Comment:
Extract line 4-line 17 to CookieNavigator.macro.

And combine with the above similar macros of CookieBannerAdmin.macro. 
e.g. Combine CookieBannerAdmin.gotoInstanceSettings with CookieConsentAdmin.gotoInstanceSettings:

macro gotoInstanceSettings {
		var baseURL = "${baseURL}";

		if (!(isSet(baseURL))) {
			var baseURL = PropsUtil.get("portal.url");
		}

		if ("${portlet}" == "Cookies Banner") {
			var portletId = "com.liferay.cookies.configuration.CookiesBannerConfiguration";
		}

		if ("${portlet}" == "Cookies Consent") {
			var portletId = "com.liferay.cookies.configuration.CookiesConsentConfiguration";
		}

		Open(locator1 = "${baseURL}/group/control_panel/manage?p_p_id=com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet&_com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet_factoryPid=${portletId}&_com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet_mvcRenderCommandName=%2Fconfiguration_admin%2Fedit_configuration");
}

Reply:
The same way to combine CookieBannerAdmin.gotoSystemSettings with CookieConsentAdmin.gotoSystemSettings.
And combine CookieBannerAdmin.gotoSiteSettings with CookieConsentAdmin.gotoSiteSettings.
Don't forget to remove the redundant parameter in urlAppend. e.g. &p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view and so on.